postgis/gserialized_gist_2d.c

```diff
@@ -1302,6 +1304,97 @@ Datum gserialized_gist_same_2d(PG_FUNCTION_ARGS)
 	PG_RETURN_POINTER(result);
 }
 
+PG_FUNCTION_INFO_V1(gserialized_gist_sortsupport_2d);
+/*
+ * Hash comparator of `GSERIALIZED` data type
+*/
+static int
+hash_cmp(Datum a, Datum b, SortSupport ssup)
```

Comment:
I'm not sure this is what the comment above says it is? It's only turned on in abbreviated mode, and then it is... not fed GSERIALIZED, but actually fed uint hash codes in the datums? (I think I'm reading what's going on below correctly, but still guessing at how the SortSupport API works.) Maybe rename things for a little more clarity.